tom_mmt/mmt.py

```diff
@@ -256,10 +256,10 @@ def observation_payload(self):
             'gain': self.cleaned_data['gain'],
             'ReadTab': self.cleaned_data['read_tab'],
             'DitherSize': self.cleaned_data['dither_size'],
-            'grism': self.cleaned_data['grism'],
+            'grism': 'J' if self.cleaned_data['filter'] == 'zJ' else 'HK',,
             'slitwidth': self.cleaned_data['slit_width'],
             'maskid': 111,
-            'filter': 'J' if self.cleaned_data['grism'] == 'zJ' else 'HK',
+            'filter': self.cleaned_data['visit'],
```

Comment:
this should be `self.cleaned_data['filter']`, right?

Reply:
That is correct. I have updated it and pushed it now.